.travis.yml

```diff
@@ -43,19 +43,13 @@ env:
   - PACKAGE='TDS_3 Three Triangulation ' 
   - PACKAGE='Triangulation_2 Triangulation_3 Union_find ' 
   - PACKAGE='Visibility_2 Voronoi_diagram_2 ' 
-  - PACKAGE='Polyhedron_demo'
-
-
-compiler: 
-  - clang-3.6
-  - gcc  
+  - PACKAGE='Polyhedron_demo' 
```

Comment:
Indeed, it is still in the template though. There seem to be a problem in the script generate_travis.sh, I'll look into it right away